packages/domain-picker/package.json

```
@@ -39,7 +39,8 @@
 		"lodash": "^4.17.15",
 		"tslib": "^1.10.0",
 		"use-debounce": "^3.1.0",
-		"uuid": "^7.0.2"
+		"uuid": "^7.0.2",
+		"@wordpress/base-styles": "^1.9.0"
```

Right now we're using `^1.8.0` elsewhere. Let's match that version.

```suggestion
		"@wordpress/base-styles": "^1.8.0"
```

---

AFAIK these are equivalent due to the caret. But I'll make them exactly the same.